module/IlluminaBeadArrayFiles.py

```diff
@@ -576,8 +576,14 @@ def normalize_intensities(self, x, y, threshold = True):
         tempx3 = tempx2 - self.shear * tempy2
         tempy3 = tempy2
 
-        xn = tempx3 / self.scale_x
-        yn = tempy3 / self.scale_y
+        if self.scale_x == 0:
+            xn = 0.0
```

Comment:
Thanks @KelleyRyanM!
Sure, making NaN is fine too if you think that makes more sense.  The main thing is that currently the function "get_normalized_intensities" would error even only one probe had zero intensity.

The manifest I used is "GSAMD-24v1-0_20011747_A1.bpm".  I'm not 100% that this is the one that the lab used to create the gtc files.  I've added 3 example gtc files and matching idats here:
https://github.com/ekarlins/BeadArrayFiles/tree/fix-zero-division/files
The manifest was too large to add to github.  Let me know if you need me to get you the manifest some other way.  The gtc files were created by Illumina LIMS using auto call.  They are version 3.  I know we can get a newer version of gtc with Beeline and AutoConvert.  Do you know if there is a way to get the newer gtc files from Illumina LIMS?

Thanks,
Eric